kobocat.patch

```diff
@@ -11,25 +11,3 @@ index 4d153b37e..61f6705b2 100644
      # via -r dependencies/pip/requirements.in
  ptyprocess==0.7.0
      # via pexpect
-@@ -293,8 +293,6 @@ requests-oauthlib==1.3.1
-     # via msrest
- s3transfer==0.5.2
-     # via boto3
--savreaderwriter @ https://bitbucket.org/fomcl/savreaderwriter/downloads/savReaderWriter-3.3.0.zip
--    # via -r dependencies/pip/requirements.in
- sentinels==1.0.0
-     # via mongomock
- sentry-sdk==1.5.10
-diff --git a/onadata/settings/base.py b/onadata/settings/base.py
-index cf2d65df6..9f89ae29d 100644
---- a/onadata/settings/base.py
-+++ b/onadata/settings/base.py
-@@ -455,7 +455,7 @@ KOBOFORM_SERVER_PORT = env.str('KOBOFORM_SERVER_PORT', '8000')
- KOBOFORM_SERVER_PROTOCOL = env.str('KOBOFORM_SERVER_PROTOCOL', 'http')
- KOBOFORM_LOGIN_AUTOREDIRECT = True
- KOBOFORM_URL = env.url('KOBOFORM_URL', 'http://kf.kobo.local').geturl()
--KOBOFORM_INTERNAL_URL = env.url('KOBOFORM_INTERNAL_URL', KOBOFORM_URL).geturl()
-+KOBOFORM_INTERNAL_URL = env.url('KOBOFORM_INTERNAL_URL', KOBOFORM_URL)
- KPI_HOOK_ENDPOINT_PATTERN = '/api/v2/assets/{asset_uid}/hook-signal/'
- 
- # These 2 variables are needed to detect whether the ENKETO_PROTOCOL should overwritten or not.
```

Comment:
No longer necessary due to  https://github.com/jnm/kobo-no-docker/commit/d32d01a77d672ae9939ba0156257f28a1c4c9d4d.